packages/playground/nested-deps/index.html

```diff
@@ -22,13 +22,16 @@ <h2>nested dependency nested-D (dep of D)</h2>
 <h2>exclude dependency of pre-bundled dependency</h2>
 <div>nested module instance count: <span class="nested-e"></span></div>
 
+<h2>absolute dependency path: <span class="absolute-f"></span></h2>
+
 <script type="module">
   import A from 'test-package-a'
   import B, { A as nestedA } from 'test-package-b'
   import C from 'test-package-c'
   import { C as sideC } from 'test-package-c/side'
   import D, { nestedD } from 'test-package-d'
   import { testExcluded } from 'test-package-e'
+  import F from '{{ F }}'
```

Comment:
Could you explain when this is valid syntax? Never saw some import like this 🤔

Reply:
It represents the absolute path of the F package and will be converted to the real path at runtime. You can look at the resolve.alias in vite.config.js.

Reply:
Ah okay, now I understand

Reply:
But i'm not sure why ci test fails in window environment.